specification/cognitiveservices/OpenAI.Inference/models/transcription.create.tsp

```diff
@@ -0,0 +1,53 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("""
+Transcription request.
+Requesting format 'json' will result on only the 'text' field being set.
+For more output data use 'verbose_json.
+""")
+model AudioTranscriptionsOptions {
+    @doc("The audio file object to transcribe.")
+    // how do we handle binary format for a member?
+    file: string;
```

Comment:
I tried the `bytes` built-in type for this and it was fairly reasonable in .NET, projecting as `BinaryData`. Questions about that vs. `Stream` for language idioms notwithstanding, I imagine `bytes` is likely a good choice -- it's certainly accurate.